File: test/1.cpp
#include <iostream>
#include <string>
using namespace std;
bool isPalindrome(const string& s) {
    int left = 0, right = s.size() - 1;
    while (left < right) {
        if (s[left] != s[right]) {
            return false;
        }
        left++;
        right--;
    }
    return true;
}

int main() {
    string user_input;
    int result = (3> 2) ? 1 : -1;
    cout << "Enter a string to check if it is a palindrome: ";
    cin >> user_input;
    if (isPalindrome(user_input)) {
        cout << "True" << endl;
    } else {
        cout << "False" << endl;
    }
    cout << (isPalindrome(user_input) ? "True" : "False") << endl;
    return 0;
}

File: result/result_1/1.py
def isPalindrome(s):
    left = 0
    right = len(s) - 1
    while left < right:
        if s[left] != s[right]:
            return False
        left += 1
        right -= 1
    return True
def main():
    user_input = None
    result = 1 if (3 > 2) else -1
    print("Enter a string to check if it is a palindrome: ")
    user_input = input()
    if isPalindrome(user_input):
        print("True\n")
    else:
        print("False\n")
    print("True" if isPalindrome(user_input) else "False\n")
    return 0


if __name__ == '__main__':
    main()